### PR TITLE
redmine:1516 integrating SFI projects reference extraction

### PR DESCRIPTION
**Pull request by marekhorst**
This change is related to project reference extraction feature described in [#1516](https://issue.openaire.research-infrastructures.eu/issues/1516) but it should also fix [#1304](https://issue.openaire.research-infrastructures.eu/issues/1304).

### CODE REVIEW

File: iis-workflows/iis-workflows-referenceextraction/src/test/java/eu/dnetlib/iis/workflows/referenceextraction/project/WorkflowTest.java [hunk 1]
```diff
@@ -19,11 +20,6 @@ public void testMainWorkflow() throws Exception {
 	}
 
     @Test
-	public void testMainSQLiteWorkflow() throws Exception{
```
Comment (mkobos) [inline iis-workflows/iis-workflows-referenceextraction/src/test/java/eu/dnetlib/iis/workflows/referenceextraction/project/WorkflowTest.java]: Out of curiosity, why is that test removed?

**Comment by mkobos** [comment]
I finished the review. Looks good to me, but I've got one question (see the comment).

File: iis-workflows/iis-workflows-referenceextraction/src/test/java/eu/dnetlib/iis/workflows/referenceextraction/project/WorkflowTest.java (hunk 1 above)
Comment (marekhorst) [inline iis-workflows/iis-workflows-referenceextraction/src/test/java/eu/dnetlib/iis/workflows/referenceextraction/project/WorkflowTest.java]: It turned out not all tests were failing due to the problem which was patched by Ioannis. 

One test was utilizing predefined sqlite db which become obsolete after minor change in projects model conducted recenty. In order to fix this test we would need to manually recreate db from scratch while in all other tests db was generated on the fly. I just found this test pointless.

Comment (mkobos) [inline iis-workflows/iis-workflows-referenceextraction/src/test/java/eu/dnetlib/iis/workflows/referenceextraction/project/WorkflowTest.java]: ok

**Pull request closed without merging**